miden-tx/src/tests/kernel_tests/mod.rs

```diff
@@ -64,53 +68,53 @@ pub fn output_notes_data_procedure(notes: &OutputNotes) -> String {
 
     format!(
         "
-    proc.create_mock_notes
```

Comment:
I haven't changed any of the MASM code behaviour, just their formatting to make it easier to read (by aligning with the rest of the code around it)